### ISSUE TITLE
@SerialName with name using dashes produces invalid JS code

### ISSUE THREAD

**Issue by flucivja** [Kotlin/kotlinx.serialization#43]
Hi,

I am playing around with multi platform project and I have defined following data class:
```
@Serializable
data class LoginRequest(
        @SerialName("user-name") var userName: String,
)
```

when I run JS build then it fails during webpack bundling due to invalid JS code (variable name contains dash which is invalid in JS):
```
Module parse failed: Unexpected token (549:12)
You may need an appropriate loader to handle this file type.
|     var index, readAll = false;
|     var bitMask0 = 0;
|     var user-name;
|     var input = input_0.readBegin_276rha$(this.serialClassDesc, []);
|     loopLabel: while (true) {
:playground-js:webpack-bundle FAILED
```

Can you check whether it is bug or whether is it possible to get around this.

Thanks,
Fero

**Comment by sandwwraith** [comment]
Yep, it's a bug. Current generation strategy translates `SerialName`s directly into JS code, turns out that it is not the best solution.

**Comment by sandwwraith** [comment]
Fixed in 0.4

**Comment by flucivja** [comment]
Nice work! Verified the fix in 0.4. Thank you for such a quick response.
